lib/class-wp-theme-json-gutenberg.php

```diff
@@ -2389,8 +2389,7 @@ function( $pseudo_selector ) use ( $selector ) {
 
 		// 3. Generate and append the rules that use the duotone selector.
 		if ( isset( $block_metadata['duotone'] ) && ! empty( $declarations_duotone ) ) {
-			$selector_duotone = static::scope_selector( $block_metadata['selector'], $block_metadata['duotone'] );
-			$block_rules     .= static::to_ruleset( $selector_duotone, $declarations_duotone );
+			$block_rules .= static::to_ruleset( $block_metadata['duotone'], $declarations_duotone );
```

Comment:
Duotone is already scoped in `get_blocks_metadata` by using `wp_get_block_css_selector`.